code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -288,6 +335,72 @@
 		filling.color = reagents.get_color()
 		add_overlay(filling)
 
+//////////////////////////JUICES AND STUFF ///////////////////////
+
+/obj/item/reagent_containers/food/drinks/carton
+	name = "carton"
+	desc = "An abstract way to organize bottles that are really cartons. Finally!"
+	item_state = "carton"
+	volume = 100
+	center_of_mass = list("x"=16, "y"=6)
+	drop_sound = 'sound/items/drop/box.ogg'
+
+/obj/item/reagent_containers/food/drinks/carton/orangejuice
+	name = "orange juice"
+	desc = "Full of vitamins and deliciousness!"
+	icon_state = "orangejuice"
+	Initialize()
```

Comment:
Could you fix this pathing? Technically out of scope so I dont mind if this is a no.

Reply:
Well it's a refactor so fuck it, yeah